proxy/grpcproxy/client.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2016 The etcd Authors
```

Comment:
i think it makes more sense to connect the client with server over network? also probably we need a better file name? client in grpcproxy pkg can be confusing.